abstract/abstract.tex

```diff
@@ -4,6 +4,34 @@
 
 \begin{document}
 
+%% Titlepage variables
+
+% Title
+\title{Instructions for Preparing an Extended Abstract for Bicycle and 
+	Motorcycle Dynamics 2023}
+
+% List authors for the title. Call \addauthor{name}{affiliationID} for every 
+% author. Authors will appear in the order off calls to \addauthor. Please 
+% manually specify the correct affiliation ID and an asterisk to the 
+% corresponding author
+\addauthor{Author B. One}{1}
+\addauthor{Author C. Two}{2}
+\addauthor{Author D. Three}{2,*}
+
+% List authors with only initials for given names, displays only in footer.
+\authorfooter{One, A. B., Two, A. C., \& Three, A. D.}
+
+% List affiliations. Call /addaffiliation{id}{name}{EmailOrchidString} once for
+% every distinct institution, where id specifies the affiliation ID (ensure 
+% that this corresponds to the IDs used with \addauthor), name specifies the 
+% affiliation name, and EmailOrchidString is a string listing emails and 
+% ORCHIDs (optional) affiliated with this affiliation separating mail and 
+% ORCHID using a comma and different author credentials using a semicolon
+
+\addaffiliation{1}{Affiliation 1}{a.b.one@affiliation1.com, ORCHID 1111111}
+\addaffiliation{2}{Affiliation 2}{a.c.two@affiliation2.com, ORCHID 2222222; 
```

Comment:
The affliation should be like:

"BioMechanical Engineering Department, Delft University of Technology; j.k.moore@tudelft.nl, ORCID 12345"

Reply:
So you are now missing the department/uni.

Reply:
Cant we just have people write that in the "Affiliation 1" part?

Reply:
And for authors from same affliation

"BioMechanical Engineering Department, Delft University of Technology; j.k.moore@tudelft.nl, ORCID 12345; c.schmidt@tudelft.nl, ORCID 12345"

Reply:
Im not sure what you mean. You could achieve that with the existing macro:

\addaffiliation{2}BioMechanical Engineering Department, Delft University of Technology}{ j.k.moore@tudelft.nl, ORCID 12345; c.schmidt@tudelft.nl, ORCID 12345}

Reply:
There is not a lot of logic in the macro. I just tried to find a quick and dirty solution to transfer the affiliation-info to maketitle. Probably you could automate this better, so that this line get build from single bits of information about authors and their affiliation, but this was a bit beyond my template skills and timeframe.

Reply:
Sorry, I misunderstood the code. Here is a real example with your code:

```
\addaffiliation{1}{Department of BioMechanical Engineering, Delft University of Technology}{j.k.moore@tudelft.nl, ORCID 1111111}
```

So that works.

Maybe we just need a note that says "Change "affliation 1" to your department, university, company, and/or organization."